articles/active-directory/develop/quickstart-v2-ios.md

```diff
@@ -88,6 +88,14 @@ In a terminal window, navigate to the folder with the downloaded code sample and
 >    ```swift
 >    let kClientID = "Enter_the_Application_Id_Here"
 >    ```
+> 1. Edit **ViewController.swift** and replace the line starting with 'let kAuthority' with the following code snippet:
+>    ```swift
+>    let kAuthority = "https://Enter_the_Cloud_Instance_Id_here/Enter_the_Tenant_Info_Here"
```

Comment:
`"Enter_the_Authority_Endpoint_Host_HereEnter_the_Tenant_Info_Here"`

The replacement for `Enter_the_Authority_Endpoint_Host_Here` includes the protocol and trailing slash, e.g. `https://login.microsoftonline.com/`.

Reply:
`Enter_the_Cloud_Instance_Id_here` is not a supported token. Please talk with @archieag if you think you need this.

Reply:
@mstada missing the slash. I believe this should be "Enter_the_Authority_Endpoint_Host_Here/Enter_the_Tenant_Info_Here"

Reply:
@archieag the first token has a trailing slash, so your suggestion would resolve with a double slash in the middle.

Reply:
@mstada, @archieag - should I update this to be 

`let kAuthority = "https://Enter_the_Cloud_Instance_Id_hereEnter_the_Tenant_Info_Here"?`

Please confirm and I'll do the updates. Thanks.

Reply:
@oldalton That's right. Please update.

Reply:
No, that's not quite right, please update to `let kAuthority = "Enter_the_Authority_Endpoint_Host_HereEnter_the_Tenant_Info_Here"`

Reply:
@oldalton I overlooked the token placeholder. @mstada is right.

Reply:
Updated.

Reply:
@odalton also please remove `https://`. It will be part of the replaced value, just like the graph endpoint.

Reply:
Done